menas/src/test/scala/za/co/absa/enceladus/menas/controllers/SchemaControllerTest.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas.controllers
+
+import org.apache.avro.SchemaParseException
+import org.apache.spark.sql.types.{DataType, DataTypes, StructField, StructType}
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest.{Inside, Matchers, WordSpec}
+import za.co.absa.enceladus.menas.controllers.SchemaController._
+import za.co.absa.enceladus.menas.models.rest.exceptions.SchemaParsingException
+import za.co.absa.enceladus.menas.utils.converters.SparkMenasSchemaConvertor
+
+import scala.io.Source
+
+class SchemaControllerTest extends WordSpec with Matchers with MockitoSugar with Inside {
```

Comment:
Usually the test classes use the suffix `Suite` not `Test` (I know there are exceptions). Let's keep a unified naming, looks more tidied 😉

Reply:
Thanks, renamed.